src/algebra/category/Algebra/limits.lean

```diff
@@ -19,6 +19,8 @@ open category_theory.limits
 
 universe u
 
+noncomputable theory
+
```

Comment:
This (and similar lines in other files) probably doesn't break anything, but it isn't necessary either.

Reply:
Removed.